_commands/tools/ffmpeg.md

```diff
@@ -2,22 +2,70 @@
 ---
 ffmpeg
 ----
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+
+ffmpeg is a multimadia framework used to record, convert, and stream audio and video.
```

Comment:
multim**e**dia!

Also could you enclose the command with backticks so it renders as `ffmpeg` and not just ffmpeg? 👍